tfac/figures/figure1.py

```diff
@@ -14,17 +15,52 @@ def makeFigure():
     """ Get a list of the axis objects and create a figure. """
     # Get list of axis objects
     ax, f = getSetup((8, 3), (1, 3))
-    ax[0].axis("off")
 
     tensor, _, _ = form_tensor()
     # perform tensor decomposition from tensorpack with 8 components
-    t = Decomposition(tensor, max_rr=6)
+    t = Decomposition(tensor, max_rr=7)
     t.perform_tfac()
     t.perform_PCA(flattenon=2)
 
-    tfacr2x(ax[1], t)
-    reduction(ax[2], t)
+    tfacr2x(ax[0], t)
+    reduction(ax[1], t)
 
+    # Scaling factors for protein dataset
+    scales = np.power(4, [-4.0,-3.0,-2.0,-1.0,0,1,2,3,4])
+    # Amount of components for tfac
+    comps = 3
```

Comment:
Sorry for getting to this late..
Is this figure being run for 3 components? I think it would be closer to our other analysis if it was for 5 components.